connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/ConnectorHelper.java

```diff
@@ -58,7 +60,10 @@ public static Map<String, Object> createOutputVariables(
 
     Optional.ofNullable(resultExpression)
         .filter(s -> !s.isBlank())
-        .map(expression -> FEEL_ENGINE_WRAPPER.evaluateToJson(expression, responseContent))
+        .map(
+            expression ->
+                FEEL_ENGINE_WRAPPER.evaluateToJson(
+                    expression, responseContent, wrapResponse(responseContent)))
```

Comment:
Why do we add the `responseContent` twice?

Reply:
In the case of a successful Connector result you would have access to the result of the evaluation of the result variable and result expression: https://github.com/camunda/connectors/blob/1003797ccc93f4526073e6451599957d03ec8b38/connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/outbound/ConnectorJobHandler.java#L176

Reply:
Yes, but in the current implementation, you can access for example the body of a rest invocation in the result expression via `response.boy` OR `body`. Passing the context twice (wrapped AND unwrapped) will ensure that this is still possible.

Reply:
Right, good point. We just need to verify that this was the only path where it was supported. Maybe it was similar for the result expressions. 

Do you know by chance @chillleader ?

Reply:
I only ever saw this in relation to result expressions and error expressions, looks like both use cases are covered